frontend/src/components/coverage/enrollment/NameSearchCandidate.vue

```diff
@@ -60,6 +61,9 @@ export default {
       if (this.candidate.dateOfBirth) {
         details = details + ' ' + this.candidate.dateOfBirth
       }
+      if (this.candidate.dateOfDeath && this.candidate.dateOfDeath != 'N/A') {
+        details = details + ' ' + this.candidate.dateOfDeath
```

Comment:
It's very hard to see that there's a date of death. If you change the template to `<span v-html="formatDetailsLine2"></span>` then you can return formatted html in the computed property. This will allow you to use <span class="text-red">.